boards/arm/96b_ivy5661/doc/index.rst

```diff
@@ -0,0 +1,235 @@
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | flash                               |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | watchdog                            |
++-----------+------------+-------------------------------------+
+
+The default board configuration can be found in the defconfig file:
+``boards/arm/96b_ivy5661/96b_ivy5661_defconfig``
+
+Connections and IOs
+===================
+
+LED
+---
+
+- LED5 / Power LED = VDD
+- LED6 / User LED  = GPIOA3
+- LED7 / User LED  = GPIOA1
+- LED7301 / User LED  = GPIOA2
+
+Push buttons
+------------
+
+- S4 / Reset Button = RST_N
+- S1 / User Button  = GPIOA0
+
+.. _Jumpers:
+
+Jumpers
+-------
+
+There are several jumpers on IVY5661, which are used for switching boot mode.
+
++--------+--------+--------+--------+----------------+-------------------------+
+| RFCTL8 | RFCTL7 | RFCTL6 | RFCTL5 |    Boot Mode   |        Description      |
++========+========+========+========+================+=========================+
+|   0    |   0    |   1    |   0    | ROM boot       | Update image from UART0 |
++--------+--------+--------+--------+----------------+-------------------------+
+|   0    |   0    |   0    |   1    | NOR Flash boot | XIP mode                |
++--------+--------+--------+--------+----------------+-------------------------+
+
+System Clock
+============
+
+96Boards IVY5661 can be driven by an internal oscillator as well as the main
+PLL clock. By default System clock is sourced by PLL clock at 416MHz, driven
+by internal oscillator.
+
+Serial Port
+===========
+
+On 96Boards IVY5661 Zephyr console output is assigned to UART0.
+Default settings are 115200 8N1.
+
+Programming and Debugging
+*************************
+
+Building
+========
+
+Here is an example for building the :ref:`hello_world` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: 96b_ivy5661
+   :goals: build
+
+Flashing
+========
+
+96Boards IVY5661 can be flashed by two methods, one using the ROM
+bootloader and another using the SWD debug port(which require
+additional hardware).
+
+96Boards IVY5661 can be flashed by two methods, one using the ROM
+bootloader and another using the SWD debug port (which requires additional
+hardware).
+
+Using ROM bootloader
+--------------------
+
+#. Connect the 96Boards IVY5661 to your host computer using the
+USB port located near 40 pin expansion header.
```

Comment:
You left off the indenting shown in my previous comments:
```
1. Connect the 96Boards IVY5661 to your host computer using the
   USB port located near 40 pin expansion header.

#. Force IVY5661 to boot into ROM code mode by setting the jumpers
   as described in Jumpers_ section.

#. Reset the IVY5661 board by pressing the RST push button located
   near the jumpers.

#. Use `uwpflash`_ utility to flash the 96Boards IVY5661:

   .. code-block:: console

      $ ./uwpflash -d /dev/ttyUSB0 -f zephyr.bin -a 0x2000000

   .. note::
      The above command assumes that IVY5661 board is connected to
      serial port ``/dev/ttyUSB0`` of the host machine.

#. Then switch the jumpers to XIP mode as described in the Jumpers_
   section and open a serial emulator:

 .. code-block:: console

    $ minicom -D /dev/ttyUSB0

#. Now press the RST button and you should see the following message on
   the console:

   .. code-block:: console

      Hello World! 96b_ivy5661
```

Reply:
Fixed.